odoo/addons/base/models/ir_actions_report.py

```diff
@@ -39,39 +41,33 @@
     pass
 
 
-def _get_wkhtmltopdf_bin():
-    return find_in_path('wkhtmltopdf')
+def _get_chrome_bin():
+    return find_in_path('google-chrome')
```

Comment:
Why not reuse `odoo.tests.common.ChromeBrowser`?

At least please do not forget to add support for chromium. :pray: